resources/user-agents/browsers/puffin/puffin-5-0-on.json

```diff
@@ -101,24 +101,28 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*IP*",
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "devices": {
             "iPhone": "iPhone",
             "iPad": "iPad"
           },
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
+            "iOS_C"
           ]
         },
         {
           "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "device": "general Mobile Device (Apple)",
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
```

Comment:
When I've been consolidating Android devices in some of the larger files, I've been rearranging the Android platform versions to be highest -> lowest, as I find it easier to scan the file and sort the child groups.

Does the lack of the zero, and the order of the pattern matter if the lack of a zero causes the pattern to be shorter anyhow (which browscap-php will move down a notch in the "priority" list when parsing)?